### ISSUE TITLE
Error: Spfx Teams app stopped working for some customers

### ISSUE THREAD

**Issue by rohannevrikar** [SharePoint/sp-dev-docs#6270]
## Category

- [x] Bug

## Describe the bug

We had deployed a Teams tab app to some customer tenants which creates meeting and returns the meeting link to the user. It had been working fine until a few weeks ago where this app crashes because of 400 (Bad request) error for some users. It is still working for some, not working for others even if in same tenant. My issue looks like a similar issue to this one [https://github.com/SharePoint/sp-dev-docs/issues/3923](url), but this issue seems to be related to authentication as some folks have talked about 403 (Forbidden) error which he faced. We checked for the same on our customer's end and generated new password credentials in the manifest of SharePoint Online Client Extensibility Web Application Principal app too, but no luck.

![image](https://user-images.githubusercontent.com/26645175/92898307-803dc100-f43b-11ea-8556-6df7ffb3c35a.png)

400 error is thrown for this call: 

`https://<tenant>.sharepoint.com/sites/TeamSupport/_api/Microsoft.SharePoint.Internal.ClientSideComponent.Token.AcquireOBOToken?resource=%27https://graph.microsoft.com%27&clientId=<clientId>
`

When I tried to reproduce the same on my end using the account which customer gave me, it was working in my end and my colleagues' too.  Unfortunately this is all the information that we have. This is failing for Teams desktop for different versions, hence maybe not a pattern here too. Can someone please help me in the right direction how to proceed with this issue? Am I missing something here?

**Comment by bcameron1231** [comment]
Looks like you have a screen shot, are you able to get a screenshot from your customer of the network tab for this call? The response may provide a bit more information as to what the 400 error is a result of. You can open the network tab in Dev Tools, select the call that's failing and then select **response** to view full response.

**Comment by rohannevrikar** [comment]
@bcameron1231 Thanks for responding. I was able to reproduce the same issue on my end, which is exactly like this one: [https://github.com/SharePoint/sp-dev-docs/issues/6136](url). If a user has not logged in to SharePoint, then this error is thrown. As soon as I logged into SharePoint from this user, it started working. I realized it was working on our end because the account which we got the customer had been used to log into SharePoint.
